pallets/phala/src/mining.rs

```diff
@@ -507,6 +504,7 @@ pub mod pallet {
 
 			let event = message.payload;
 			if !event.is_empty() {
+				let emit_ts = event.timestamp_ms;
 				let now = Self::now_sec();
```

Comment:
I don not fully understand the logic in this PR. But be careful the units of `timestamp_ms` and  is `Self::now_sec` are differrent.

Reply:
Fixed